File: NetworkProject-Unity/Assets/Scripts/Class/Player.cs
using UnityEngine;

public class Player
{
    public string Name;
    public Vector3 Position;
    public int Type;

    public Player(){}
    public Player(string name, Vector3 position, int type){
        this.Name = name;
        this.Position = position;
        this.Type = type;
    }
}

File: WebSocket-python/player.py
import json
from typing import Tuple

class Player:
    Name = ""
    Position = {}
    Type = 0

    def __init__(self, Name: str, Position: Tuple, Type: int):
        self.Name = str(Name)
        self.Position = Position
        self.Type = Type
    
    def toJSON(self):
        return json.dumps(self, default=lambda o: o.__dict__, sort_keys=True, indent=4)